saleor/graphql/webhook/subscription_types.py

```diff
@@ -878,6 +881,35 @@ class Meta:
         interfaces = (Event,)
 
 
+class PermissionGroupBase(AbstractType):
+    permission_group = graphene.Field(
+        "saleor.graphql.account.types.Group",
+        description="The permission group the event relates to."
+        + ADDED_IN_35
+        + PREVIEW_FEATURE,
+    )
+
+    @staticmethod
+    def resolve_permission_group(root, _info):
+        _, permission_group = root
+        return permission_group
+
+
+class PermissionGroupCreated(ObjectType, PermissionGroupBase):
+    class Meta:
+        interfaces = (Event,)
+
+
+class PermissionGroupUpdated(ObjectType, PermissionGroupBase):
+    class Meta:
+        interfaces = (Event,)
+
+
+class PermissionGroupDeleted(ObjectType, PermissionGroupBase):
+    class Meta:
+        interfaces = (Event,)
```

Comment:
Lack of descriptions.

Reply:
We'll add all missing descriptions in one PR.